src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/notification/NotificationRequest.java

```diff
@@ -53,4 +53,6 @@ public class NotificationRequest {
     private String intervenerSolicitorFirm;
     @JsonProperty("")
     private byte[] documentContents;
+    @JsonProperty("isNotDigital")
+    private Boolean isNotDigital;
```

Comment:
Can this be a primitive boolean rather than an object? What happens if it is null?

Reply:
We have to use the object rather than the primitive to work with class annotations "@Data and @Builder" annotations. These annotation I believe will set the boolean to false if not provided.

Reply:
The lombok annotations @Data and @Builder just create methods. They don't assign values.
If you have situations where isNotDigital can be null then Boolean is appropriate otherwise the primitive would seem to make more sense.